openhtf/__init__.py

```diff
@@ -37,11 +38,35 @@
 
 FLAGS = gflags.FLAGS
 FLAGS(sys.argv)
-
+gflags.DEFINE_string('log_dir', '/tmp', 'dir for log files')
+gflags.DEFINE_enum('log_output', 'console', ['console','file','both'], 'output option')
```

Comment:
For controlling stdout verbosity I suggest using the -v verbosity convention where:
default: send only critical and error level messages to stdout
-v: send warning and above level messages to stdout
-vv: send info and above level messages to stdout
-vvv: send everything to stdout